app/src/main/res/layout/fragment_home.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<androidx.constraintlayout.motion.widget.MotionLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        xmlns:tools="http://schemas.android.com/tools"
+        android:id="@+id/homeLayout"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        app:layoutDescription="@xml/home_scene"
+        tools:context=".home.HomeFragment"
+        android:clipToPadding="false">
+
+
+    <ImageButton
+            android:id="@+id/menuButton"
+            android:src="@drawable/ic_menu"
+            android:layout_width="48dp"
```

Comment:
We may want to specify some of these hardcoded sizes in dimens.xml, so we can support tablets later.